app/workers/find_and_suspend_inactive_accounts_worker.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class FindAndSuspendInactiveAccountsWorker
```

Comment:
Can the worker be called `SuspendInactiveAccountsWorker`?

Reply:
https://github.com/3scale/porta/pull/17#discussion_r229264561 😄